### PR TITLE
HDDS-2827. Add initial UI of Datanodes in Recon

### PR DESCRIPTION
**Pull request by vivekratnavel**
## What changes were proposed in this pull request?

New addition of UI page for listing Datanodes in Recon web app.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-2827

## How was this patch tested?

This patch was tested by running the react app with mock api server.
```
cd hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web
yarn install
yarn run dev
```

A screen-shot of the new Datanodes page is attached below:
Screen Shot 2020-01-07 at 10 06 45 PM

### CODE REVIEW

**Comment by vivekratnavel** [comment]
@elek @avijayanhwx Please review

**Comment by avijayanhwx** [comment]
@vivekratnavel The UI looks good to me. Does this table widget currently include sorting by a specific column's values?

**Comment by vivekratnavel** [comment]
@avijayanhwx Yes. Sorting can be enabled anytime. It is disabled for time being since we need to eventually support server side sort, filter and pagination.

File: hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/Datanodes/Datanodes.tsx [hunk 1]
```diff
@@ -17,16 +17,171 @@
...
-      <div>
-        <div className="page-header">
-          Datanodes
-        </div>
-        <div className="content-div">
-          Datanodes content
-        </div>
-      </div>
-  );
+export type DatanodeStatus = "HEALTHY" | "STALE" | "DEAD" | "DECOMMISSIONING" | "DECOMMISSIONED";
```
[the remark this answers is not in the document]
Comment (vivekratnavel) [reply hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/Datanodes/Datanodes.tsx]: Thanks @elek for the information. I will keep track of HDDS-1880-Decom

**Pull request closed without merging**